frontend/packages/console-dynamic-plugin-sdk/src/coderefs/__tests__/coderef-resolver.spec.ts

```diff
@@ -305,10 +340,14 @@ describe('resolveExtension', () => {
     };
 
     expect(isCodeRefError(errorCodeRef)).toBe(false);
-    expect(await resolveExtension(extension)).rejects.toBeTruthy();
-    expect(isCodeRefError(errorCodeRef)).toBeTruthy();
+    await resolveExtension(extension);
+    expect(isCodeRefError(errorCodeRef)).toBe(true);
+    expect(getCodeRefError(errorCodeRef)).toEqual(new Error());
     expect(errorCodeRef).toHaveBeenCalledTimes(1);
-    expect(await resolveExtension(extension)).rejects.toBeTruthy();
+
+    await resolveExtension(extension);
+    expect(isCodeRefError(errorCodeRef)).toBe(true);
+    expect(getCodeRefError(errorCodeRef)).toEqual(new Error());
```

Comment:
@christianvogt Can you please check this change? Your assertion checks that `resolveExtension` returns a Promise rejection, but for me it looks like that it returns the data and only isCodeRefError and getCodeRefError returns the error. Is this expected? Is this change (which touches only the spec file) fine for you?

Reply:
It's sufficient to assert `expect(isCodeRefError(errorCodeRef)).toBe(true);`

You can remove these checks: `expect(getCodeRefError(errorCodeRef)).toEqual(new Error());`

Reply:
Christian also agreed on slack that the change is fine. Removed assertions for `getCodeRefError`.